<launch>
	<param name="use_sim_time" value="true"/>
	<node name ="obst_detect_test" pkg = "obstacle_tracking" type = "obst_detect"/>
	<node name ="obst_track_test" pkg = "obstacle_tracking" type = "obst_track" output="screen"/>
</launch>
